.circleci/scripts/check_for_casetwins.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -euo pipefail
+shopt -s globstar nullglob
```

Comment:
I wouldn't add these as a matter of course. I added them because I was actually using them. Nullglob especially (as we've seen in the past) can cause issues.

Reply:
Yep - that was mindless copying.